### DISCUSSION TITLE
Welcome to dotty-cps-async Discussions!

### DISCUSSION THREAD

**Discussion by rssh** [rssh/dotty-cps-async#36] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by marrious11** [comment]
> Hi sir @rssh, I am Soh, from Silicon Mountain, Buea. 

I am interested in metaprogramming, and functional programming as a whole. The scalacenter community is actually my second encounter with open source contribution, and my very first programming with scala begins this April 2023.
So excited to have heard of scala while exploring functional languages, and through GSOC.

- [x] I would love to foster a deep grasp about what **[dotty-cps-async](https://github.com/rssh/dotty-cps-async)** is all about, and how I can implement them in the real world and also provide resolution to issues regarding it.
